projects/packages/forms/src/blocks/contact-form/components/jetpack-manage-responses-settings.js

```diff
@@ -23,15 +19,6 @@ const JetpackManageResponsesSettings = ( {
 					{ __( '(opens in a new tab)', 'jetpack-forms' ) }
 				</span>
 			</Button>
-			{ /* Temporarily hiding the Form Title */ }
```

Comment:
Remove this altogether. A title should be defined in the markup and be visible to all users.

Reply:
Yes, we should be good by removing it. Not sure how long ago that temporarily was there :)